gitbook-docs/market/economics.md

```diff
@@ -16,109 +18,164 @@ Proof Market constants:
 | _app\_dev_ | 0.7   | application developers part of the fee |
 | _cir\_dev_ | 0.3   | circuit developers part of the fee     |
 
-### Basic Workflow
+### Basic workflow
 
-1. A proof buyer sends a bid order to the market with a cost _c\_b_;
-2. Proof Market locks _c\_b_ tokens of the buyer;
-3. A proof producer sends an ask order to the market with a cost _c\_a <= c\_b_;
-4. Proof Market matches bid and ask order;
-5. The proof producers generate a proof and send it to the market;
-6. Proof Market verifies proof and pays _c\_b - commission_ tokens to the producer;
-7. The proof buyer uses takes their proof and uses it in some way.
+1. A proof buyer sends a request with a desired price _c\_b_ to the market.
+2. The Proof Market locks _c\_b_ tokens from the buyer's account.
+3. A proof producer sends a proposal to the market with a price _c\_a <= c\_b_.
+4. The Proof Market matches the request with the proposal.
+5. The proof producers generate a proof and send it to the market.
+6. The Proof Market verifies proof and pays _c\_b - commission_ tokens to the producer.
+7. The proof buyer takes their proof and uses it.
```

Comment:
Now that we've renamed bids and asks, the variables should be renamed as well. cost_ask → cost_request (c_r).

Reply:
same with time_ask and time_bid

Reply:
@NickVolynkin 
> cost_ask → cost_request (c_r)

You mean cost_ask → cost_proposal? And request instead of bid

Reply:
Yes!